wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -196,6 +197,15 @@ public void onChanged(de.schildbach.wallet.data.BlockchainState blockchainState)
         });
     }
 
+    @Override
+    public void onStart() {
+        super.onStart();
+
+        if (!getLockScreenDisplayed() && config.getShowNotificationsExplainer()) {
+            explainPushNotifications();
+        }
```

Comment:
If it's a new install, the explainer dialog is shown when user ends up on the home screen.